### ISSUE TITLE
Taskbar issue

### ISSUE THREAD

**Issue by TaakoOfficial** [lunar-os/windowsdesktop#2]
I copied your exact Wind hawk settings but my task bar just doesn't look the same

**Comment by Fazented** [comment]
yeah, same for me. The only part that works is the clock has the coloured background. None of the icons have outlines or the coloured background though

**Comment by OD728** [comment]
I'm missing the background for the clock/task icon area
![image](https://github.com/user-attachments/assets/8557a297-fe80-40d3-b4bd-3103432d01ae)

**Comment by Somebodyoncetoldhim** [comment]
Hey, i copied everything from your wind hawk configs but it does not work at all for me, any ideas why?

**Comment by lunar-os** [comment]
> yeah, same for me. The only part that works is the clock has the coloured background. None of the icons have outlines or the coloured background though

your using the outdated one, try the new one in my new repo

**Comment by lunar-os** [comment]
> Hey, i copied everything from your wind hawk configs but it does not work at all for me, any ideas why?

try my new updated repo

**Comment by Somebodyoncetoldhim** [comment]
> > Hey, i copied everything from your wind hawk configs but it does not work at all for me, any ideas why?
> 
> try my new updated repo

What about these icons that u use? How do i use/open them?
